crates/typst/src/geom/gradient.rs

```diff
@@ -0,0 +1,1089 @@
+///
+/// ✅ This color map is suitable for data visualization.
+///
+/// ```example
+/// #rect(width: 100pt, height: 20pt, fill: gradient.linear(..gradient.magma))
+/// ```
+///
+/// ### Plasma
+///
+/// The [`plasma`]($gradient.plasma) gradient is a purple to pink to yellow
+/// gradient that is perceptually uniform. Plasma does not take any parameters.
+///
+/// ✅ This color map is suitable for data visualization.
+///
+/// ```example
+/// #rect(width: 100pt, height: 20pt, fill: gradient.linear(..gradient.plasma))
+/// ```
+///
+/// ### Rocket
+///
+/// The [`rocket`]($gradient.rocket) gradient is a black to red to white
+/// gradient that is perceptually uniform. Rocket does not take any parameters.
+///
+/// ✅ This color map is suitable for data visualization.
+///
+/// ```example
+/// #rect(width: 100pt, height: 20pt, fill: gradient.linear(..gradient.rocket))
+/// ```
+///
+/// ### Mako
+///
+/// The [`mako`]($gradient.mako) gradient is a black to teal to yellow
+/// gradient that is perceptually uniform. Mako does not take any parameters.
+///
+/// ✅ This color map is suitable for data visualization.
+///
+/// ```example
+/// #rect(width: 100pt, height: 20pt, fill: gradient.linear(..gradient.mako))
+/// ```
+///
+/// ### Vlag
+///
+/// The [`vlag`]($gradient.vlag) gradient is a light blue to white to red
+/// gradient that is perceptually uniform. Vlag does not take any parameters.
+///
+/// ✅ This color map is suitable for data visualization.
+///
+/// ```example
+/// #rect(width: 100pt, height: 20pt, fill: gradient.linear(..gradient.vlag))
+/// ```
+///
+/// ### Icefire
+///
+/// The [`icefire`]($gradient.icefire) gradient is a light teal to black to yellow
+/// gradient that is perceptually uniform. Icefire does not take any parameters.
+///
+/// ✅ This color map is suitable for data visualization.
+///
+/// ```example
+/// #rect(width: 100pt, height: 20pt, fill: gradient.linear(..gradient.icefire))
+/// ```
+///
+/// ### Flare
+///
+/// The [`flare`]($gradient.flare) gradient is an orange to purple gradient
+/// that is perceptually uniform. Flare does not take any parameters.
+///
+/// ✅ This color map is suitable for data visualization.
+///
+/// ```example
+/// #rect(width: 100pt, height: 20pt, fill: gradient.linear(..gradient.flare))
+/// ```
+///
+/// ### Crest
+///
+/// The [`crest`]($gradient.crest) gradient is a blue to white to red gradient
+/// that is perceptually uniform. Crest does not take any parameters.
+///
+/// ✅ This color map is suitable for data visualization.
+///
+/// ```example
+/// #rect(width: 100pt, height: 20pt, fill: gradient.linear(..gradient.crest))
+/// ```
+///
+/// ### Regarding presets like "jet" and "parula"
+///
+/// - [Jet](https://jakevdp.github.io/blog/2014/10/16/how-bad-is-your-colormap/)
+///   is not a good color map, as it is not perceptually uniform. As such,
+///   it is not color blind friendly and should not be used for data visualization,
+///   due to which it is not included in Typst.
```

Comment:
```suggestion
///   it is not color blind friendly and should not be used for data visualization;
///   thus, it is not included in Typst.
```

Reply:
well, both are correct
but I'm ok with changing it

Reply:
If both are correct then I see no reason to change and drag this PR down any more.